zeppelin-web/src/components/noteName-import/notenameImport.controller.js

```diff
@@ -110,4 +115,27 @@ angular.module('zeppelinWebApp').controller('NoteImportCtrl', function($scope, $
     vm.resetFlags();
     angular.element('#noteImportModal').modal('hide');
   });
+
+  $scope.$on('importNoteFail', function(event, data) {
+    vm.resetFlags();
+    angular.element('#noteImportModal').modal('hide');
+    var status = data.importStatus;
```

Comment:
One comment for all the code below:
* Since status is used once in a if, I don't think there is a need to assign its value to a variable
* There is a bit of repetition in the ngToast configuration, creating a config object before the if/else would limit the code:

```
var ngToastConf = {
  content: 'Failed to import Note with hash',
  verticalPosition: 'bottom',
  horizontalPosition: 'center',
  timeout: '5000'
};

if (data.importStatus === 'success') {
  ngToastConf.content = 'Successfully imported Note';
  ngToast.success(ngToastConf);
} else {
  ngToast.danger(ngToastConf);
}
```

* The Note name should probably be shown in the ngToast Message